src/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -322,6 +322,11 @@ public static class MsalError
         /// </summary>
         public const string NoRedirectUri = "no_redirect_uri";
 
+        /// <summary>
+        /// No Redirect URI.
+        /// </summary>
+        public const string B2CHostNotTrusted = "B2C_host_not_trusted";
```

Comment:
>B2CHostNotTrusted [](start = 28, length = 17)

Probably a different PR... but I believe JM wanted these to be on the exceptions (in this case that would be on the clientexception),

Reply:
[this](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1004) definitely separate PR, as i want to get this out to customer, but i'll sync w/JM on this.

---
In reply to: [270208745](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1017#discussion_r270208745) [](ancestors = 270208745)